rfb/rfbclient.h

```diff
@@ -378,6 +379,12 @@ typedef struct _rfbClient {
 	/** The TLS session for Anonymous TLS and VeNCrypt */
 	void* tlsSession;
 
+	/**
+	 * Mutex to protect concurrent TLS read/write.
+	 * For internal use only.
+	 */
+	MUTEX(tlsRwMutex);
```

Comment:
please make sure this goes to the end of the struct to keep ABI compatibility

Reply:
Done!